recipe/recipe.yaml

```diff
@@ -34,61 +38,110 @@ requirements:
     - if: win
       then: getopt-win32
 
+
 tests:
```

Comment:
This can be simplified and a lot of the duplication can be dropped: https://rattler.build/latest/testing/#testing-package-contents